osf/models/metaschema.py

```diff
@@ -75,6 +75,7 @@ def __unicode__(self):
 
 class RegistrationSchema(AbstractSchema):
     config = DateTimeAwareJSONField(blank=True, default=dict)
+    description = models.TextField(null=True, blank=True)
```

Comment:
high-level description for the schema stored here - (original version of this work mapped description to config.tooltip)